gcp_variant_transforms/beam_io/vcfio.py

```diff
@@ -47,6 +48,10 @@
 #   - 'G': one value for each possible genotype.
 #   - 'R': one value for each possible allele (including the reference).
 VariantInfo = namedtuple('VariantInfo', ['data', 'field_count'])
+# Stores data about failed VCF record reads. `line` is the text line that
+# caused the failed read and `file` is the name of the file that the read
+# failed in.
+MalformedVcfRecord = namedtuple('MalformedVcfRecord', ['file', 'line'])
```

Comment:
nit: s/file/file_name (file may refer to actual file handle).

Reply:
Done.